students/Sahlberg/Lesson3/mailroom.py

```diff
@@ -0,0 +1,90 @@
+"""Ian Sahlberg
+Assignment 3 mailroom
+Python 210
+12/28/2018"""
+
+
+#Disctionary of donors name, dollar amount
+donor = {'bob johnson': [150.00],
+         'susan skoosan': [2000.00, 550.00],
+         'tim tam':[10.50],
+         'roxanne raffle':[13.00, 75.00, 123.00],
+         'jon jacob':[5000.00, 5.00]
+         }
+
+def donate(name, dictionary):
+    """Appends a donation amount to a given donor."""
+    amount = int(input('How much did they donate?'))
+    return amount, dictionary[name].append(amount)
+
+def print_list(dictionary):
+    """Prints our the donor list"""
+    return print(list(dictionary.keys()))
+
+def user_options():
+    """Main user options menu choices."""
+    user_action = input('Please choose from the following options: Send a Thank You, Create a Report, quit')
+    return user_action
+
+def email(name, amount):
+    """Create email for specific donor and amount donated."""
+    return print(f'\n\nThank you {name.title()} for the generous donation of $ {amount:.2f}. We appreciate your generosity.\n\nSincerely, \n\nThe Helping R Us Team\n\n')
+
+def quit():
+    """Quit the program"""
+    return print('Until next time!')
+
+def sort_and_stats_dict(dictionary):
+    """Create a stats dictionary from a donor dictionary and sort by total donated."""
+    pre_sort = {k: [sum(v), len(v), round((sum(v) / len(v)), 0)] for k, v in dictionary.items()}
+    print(pre_sort)
+    donor_sorted = sorted(list(pre_sort.items()), key=lambda x: x[1], reverse=True)
+    return donor_sorted
+
+def print_report(dictionary):
+    """Print out a report of donors and amounts donated, including stats (total donated, number of times donated, average donation)."""
+    print('Donor Name       |      Donor Total    |  Count of Donations    |  Average of Donations')
+    print('----------------------------------------------------------------------------------------')
+    count = 0
+    for donor in dictionary:
+        print(
+            f'{dictionary[count][0]:<20}$    {dictionary[count][1][0]:<23.2f}{dictionary[count][1][1]:<15}    ${dictionary[count][1][2]:>12.0f}')
+        count += 1
+
+if __name__== '__main__':
+    user_action = user_options()
+    if user_action.lower() == 'quit':
+        quit()
+
+    #Add users and donations, send thank you email when done.
+    while user_action == 'Send a Thank You':
```

Comment:
You can do a lot to make this a bit more user friendly, like while user_action is equal to lowercase 's' for example, so the user can input 'Send' 'send' 'S' or 's' for example.